foundationdb/examples/class-scheduling.rs

```diff
@@ -70,16 +70,16 @@ fn all_classes() -> Vec<String> {
 }
 
 fn init_classes(trx: &Transaction, all_classes: &[String]) {
-    let class_subspace = Subspace::from("class");
+    let class_subspace = Subspace::from(&"class");
```

Comment:
I needed to add that. `Subspace::from` may not be properly specified, I'll look into it.

Reply:
Maybe it works only on nightly compiler.. I didn't thought it might not work on stable because my default toolchain is a nightly.. Please ignore this comment, sorry for the noise.

Reply:
Ok, I just double checked... the CLI cargo build works fine. Apparently it's the VSCode plugin that is reporting an error when it is not. Odd.